Core/Core/Inbox/View/InboxView.swift

```diff
@@ -115,6 +119,27 @@ public struct InboxView: View, ScreenViewTrackable {
         }
     }
 
+    @ViewBuilder
+    private func unarchiveButton(message: InboxMessageListItemViewModel) -> some View {
+        if message.state == .archived {
+            Button {
+                model.updateState.send((messageId: message.id, state: .read))
+            }
+            label: {
+                Label {
+                    Text("Unarchive", bundle: .core)
+                } icon: {
+                    Image.unarchiveLine
+                        .foregroundColor(.textLightest)
+                }
+                .labelStyle(.iconOnly)
+            }
+            .tint(.ash)
+        } else {
+            SwiftUI.EmptyView()
+        }
```

Comment:
nit: Do we need the `else` in this case? Won't the `@ViewBuilder` do the same if we omit it?

Reply:
I'm not sure, this code is copy-paste from the already existing archive button.